### ISSUE TITLE
Remove comment

### ISSUE THREAD

**Issue by mathieutu** [prettier/plugin-php#113]
I think this one can be prettier hard, but Prettier should not remove comment.

```diff
-    protected $policies = [
-        // 'App\Model' => 'App\Policies\ModelPolicy',
-    ];
+    protected $policies = [];
```

**Comment by evilebottnawi** [comment]
@mathieutu known issue, right now parser don't respect comments in array, you can help with this in https://github.com/glayzzle/php-parser/issues/97

**Comment by mgrip** [comment]
Fixed with #265
